daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -211,6 +206,37 @@ private[lf] final class Compiler(
       case None => expr
     }
 
+  @inline
+  private[this] def app(f: SExpr, a: SExpr) = SEApp(f, Array(a))
+
+  @inline
+  private[this] def let(bound: SExpr)(body: Position => SExpr) =
+    SELet1General(bound, body(nextPosition()))
```

Comment:
Is it better to call `SELet1` ?

Reply:
Do we now use `SELet` anywhere? Can we drop it?

Reply:
Why not `SELet` ? in any case on another PR.

Reply:
@dajmaki we will drop the SELet in another PR.